docker-compose.yml

```diff
@@ -67,10 +72,7 @@ services:
       - ./src/workflow/icat_activemq.xml:/opt/activemq/conf/activemq.xml
 
   autoreducer:
-    build: https://github.com/neutrons/post_processing_agent.git#main
-    hostname: autoreducer
-    volumes:
-      - ./test/data:/SNS/
+    image: rosswhitfield/postprocessing:latest
```

Comment:
Is this the "right place" for this image to reside?

Reply:
Oh whoops, I accidentally committed that since I was having issues running the autoreducer locally